tox.ini

```diff
@@ -30,7 +30,7 @@ deps =
     pytest7: --requirement deps/test-old.txt
 commands =
     pip install --editable .
-    pytest {posargs:-n auto}
+    pytest {posargs:-n auto} -vvv
```

Comment:
stray debug code?  You should be able to call `tox -e test -- -vvv` iirc.

Reply:
Trying to track down an infinite-hanging test on CI, but I think I've narrowed down *which* test is causing this at least.